src/server/utils/config.ts

```diff
@@ -19,6 +19,15 @@ export const WG_ENV = {
   PORT: assertEnv('PORT'),
   /** If IPv6 should be disabled */
   DISABLE_IPV6: process.env.DISABLE_IPV6 === 'true',
+  /** Override automatic detection */
+  AWG:
+    process.env.AWG === 'true'
+      ? true
+      : process.env.AWG === 'false'
```

Comment:
currently if a variable is not set it defaults to a default value.
here not setting it has its own meaning. therefore i think it should be named something like `OVERRIDE_AUTO_AWG` or something. can't think of a better name rn